cli/aws_orbit/models/context.py

```diff
@@ -561,6 +565,57 @@ def fetch_toolkit_data(context: V) -> None:
         top_level = "orbit" if isinstance(context, Context) else "orbit-f"
 
         resp_type = Dict[str, List[Dict[str, List[Dict[str, str]]]]]
+
+        slrt_toolkit_name: str = "orbit"
+        slrt_toolkit_stack_name: str = f"softwarelabs-remote-toolkit-{slrt_toolkit_name}"
+        slrt_prefix: str = f"slrt-{slrt_toolkit_name}"
+
+        try:
```

Comment:
all of this can be replaced with the cfn module from the remote toolkit:
```
from softwarelabs_remote_toolkit import cfn
stack_name = cfn.get_stack_name(slrt_toolkit_name)
stack_exists, stack_outputs = cfn.does_stack_exist(stack_name)
```
stack_exists is a bool, stack_outputs is Dict with key/value pairs of the stack outputs